content/docs/command-reference/params/index.md

```diff
@@ -36,6 +36,9 @@ stages:
           - batch_size
 ```
 
+💡 Parameters can also be used directly in `dvc.yaml` itself. See the
```

Comment:
It's too abrupt here, in this context  `can also be used directly in `dvc.yaml` itself` is very hard to read properly.